qa/L0_model_config/autofill_noplatform_success/custom/no_backend.identity/expected

```diff
@@ -31,3 +31,4 @@ output_pinned_memory {
 }
 }
 backend: "identity"
+runtime: ""
```

Comment:
Why does `onnxruntime` expect `libtriton_onnxruntime.so`, but `identity` doesn't expect `libtriton_identity.so`?

Reply:
It is because autocomplete can only fill in "known" backends, while onnx is a "known" backend and identity is a custom backend.
See: https://github.com/triton-inference-server/core/pull/285/files#diff-f947d03afcf01249667bedc5b955404da083564c22809b6666ba923356f7c829R977
Therefore, the runtime field for identity backend is left empty on autocomplete, if not provided, until filled in later when the model is loaded.

Reply:
I see, thanks